src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { PreparedPosts } from '../../type/type';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+import './PostInfo.scss';
+
+interface Props {
+  postInfo: PreparedPosts;
+}
+
+export const PostInfo: React.FC<Props> = ({ postInfo }) => {
+  const {
+    title, body, user, comments, id,
+  } = postInfo;
+
+  return (
+    <div className="post-info">
+      <h3>
+        {`Post #${id}: ${title}`}
+      </h3>
+      <p>{body}</p>
+      {user && (
+        <div>
+          <p><UserInfo userInfo={user} /></p>
+          <p><CommentList commentList={comments} /></p>
```

Comment:
Коментарі показуємо незалежно від того, чи є користувач чи його немає.
І треба додатково вирішити той випадок, коли `comments` масив буде пустим
```suggestion
         <CommentList commentList={comments} />
```

Reply:
Пофіксила